### ISSUE TITLE
SCCM silent installation with no update

### ISSUE THREAD

**Issue by wrfortiscue** [nvaccess/nvda#9256]
Hello,

Trying to fully automate the install while having no update box popup and reminder to submit information box.

I could not find a registry setting to remove updates. I did however edit the settings and got an INI file. I can drop this into the appdata directory of the current user and it works, but only when manually running the install script.

No matter how I setup the install in SCCM, it does not recognize the nvda folder in appdata with ini settings until I close the nvda application and re open it.

I have to install the application "for system" in sccm for admin rights, then I also have a dependency "as user" to install the appdata folder.

so long story short I can't get rid of the initial update/information popup box and I would like to remove it.

Thank you

**Comment by ruifontes** [comment]
What NVDA version are you trying to install?

I have a customer also trying to do a completly automatized and silent 
install in a network...

Best regards,

Rui Fontes
NVDA portuguese team


Às 00:01 de 09/02/2019, wrfortiscue escreveu:

**Comment by leonardder** [comment]
I've also got some requests to make it possible to disable the update check system wide. A working approach could be enforcing the start of NVDA in secure mode (with the --secure) flag, but this is far from ideal.

See also #8574 which added the ability to disable start at logon with a command line parameter.

**Comment by wrfortiscue** [comment]
2018.3.2 Thanks

**Comment by wrfortiscue** [comment]
Hello,

Any update to this?

Thank you

**Comment by leonardder** [comment]
Not yet. I'll try to discuss this with either @michaeldcurran or @feerrenrut in the near future.

**Comment by wrfortiscue** [comment]
Thank you

**Comment by Adriani90** [comment]
cc: @gerald-hartig, @seanbudd for triaging.

**Comment by gerald-hartig** [comment]
In what way does the `--install-silent` flag not address this? https://www.nvaccess.org/corporate-government/#deployment

**Comment by Adriani90** [comment]
Indeed I think that parameter should be what was expected here. I am closing as works for me.
If you are still having this issue, please comment and we can reopen.
